resources/assets/less/bem/osu-layout.less

```diff
@@ -174,6 +196,16 @@
         ._width-desktop(-@spacing / 2);
       }
     }
+
+    &--landing {
+      max-width: @landing-width;
+    }
+
+    &--landing-sitemap {
```

Comment:
might as well do this in `.landing-sitemap` block itself and skip using `osu-layout` elements.